docs/developer-guide/local-development-gotchas.md

```diff
@@ -1,7 +1,30 @@
-# Local development gotchas
+## How do I disable Django's 'debug' mode?
 
-This page lists common local development issues and how do resolve them.
+Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:
 
-## Brief description of problem
+1. Add `DEBUG = False` to `config/settings/local.py` 
+2. Add `DEBUG=False` to your `.env` file
```

Comment:
Try adding `[TOC]` here instead for functional jump/skip llnks

Reply:
We decided not to keep it at this time.